src/common/transformations/include/ov_ops/rms.hpp

```diff
@@ -43,6 +43,12 @@ class TRANSFORMATIONS_API RMS : public ov::op::Op {
         m_epsilon = epsilon;
     }
 
+    void set_output_type(const element::Type& output_type) {
```

Comment:
Rename this memeber to avoid masking base class member then 
`using Node::set_output_type`;

should not be requried.

Reply:
I don't think that `set_rms_output_type` is a good choice as it duplicates class name it it's method. 
As I can see, we have 2 approaches used in other ops: `set_output_type` + using (ShapeOf, NMS, Range, etc) and `set_output_type_attr` (NMSRotated, SearchSorted). 

Seems like the second one is used in latest PRs, so please rename it to `set_output_type_attr`